torchtune/models/flamingo/_transform.py

```diff
@@ -27,7 +27,7 @@ class FlamingoTransform(ModelTokenizer, Transform):
 
     Args:
         path (str): Path to pretrained tiktoken tokenizer file.
-        tile_size (int): Size of the tiles to divide the image into. Default 224.
+        tile_size (int): Size of the tiles to divide the image into.
```

Comment:
is that true? did we remove the default?

Reply:
yep, this is a required argument